LICENSE

```diff
@@ -1,190 +0,0 @@
-                                 Apache License
```

Comment:
Re-Add

Reply:
Also this as well, sorry left un-staged as it was a new file